### PR TITLE
Add set_geom_points_and_edges() function

### PR DESCRIPTION
**Pull request by trhille**
Add set_geom_points_and_edges() function to compass.landice.mesh module, and call from build_cell_widths(). This will likely be used by all landice mesh-generating test cases.

### CODE REVIEW

File: compass/landice/mesh.py [hunk 1]
```diff
@@ -60,3 +61,43 @@ def gridded_flood_fill(field):
         lastSearchList = newSearchList
 
     return floodMask
+
+
+def set_geom_points_and_edges(xx0, xx1, yy0, yy1):
+    """
+    Set node and edge coordinates to pass to build_planar_mesh().
```
Comment (xylar) [inline compass/landice/mesh.py:68]:
```suggestion
    Set node and edge coordinates to pass to 
    :py:func:`mpas_tools.mesh.creation.build_mesh.build_planar_mesh()`.
```
If you do this, it should automatically link to that function in the MPAS-Tools API, which can add clarity and make it easier to navigate.

**Review by xylar — APPROVED** [review]
This looks good to me!

The names `xx0`, `xx1`, etc. are a little odd but there's presumably a history to that.

I have a minor suggestion for supporting cross-linking between the documentation for MPAS-Tools and for this package, but it's fine either way.

File: compass/landice/mesh.py (hunk 1 above)
Comment (trhille) [reply compass/landice/mesh.py]: Oh great, thanks!

**Comment by trhille** [comment]
> This looks good to me!
> 
> The names `xx0`, `xx1`, etc. are a little odd but there's presumably a history to that.
That is what they were called in the legacy code, but I could call them xMin, xMax, yMin, yMax for clarity.

**Comment by xylar** [comment]
> That is what they were called in the legacy code, but I could call them xMin, xMax, yMin, yMax for clarity.

I'd be very happy with `x_min`, etc. or `xmin`, etc.  We don't use camel case for python variables.

**Review by xylar — APPROVED** [review]
Perfect!

**Comment by xylar** [comment]
@trhille, could you rebase onto `master` now that #320 is in?

**Comment by trhille** [comment]
> @trhille, could you rebase onto `master` now that #320 is in?

Rebased on force-push to https://github.com/MPAS-Dev/compass/commit/a09ebb1f7d34e65f736eed10fa9e453f6f8cf30c

File: compass/landice/mesh.py [hunk 2]
```diff
@@ -60,3 +61,44 @@ def gridded_flood_fill(field):
         lastSearchList = newSearchList
 
     return floodMask
+
+
+def set_geom_points_and_edges(xmin, xmax, ymin, ymax):
```
Comment (matthewhoffman) [inline compass/landice/mesh.py:66]: I propose to rename this to `set_rectangular_geom_points_and_edges` or similar.  I suspect defining the region to mesh by a rectangle is what we'll always do for convenience, but in principle we could create any custom polygon shape to mesh inside of.  When we get to ultra high res or possibly if we try to mesh to exact boundaries, we may introduce other, similar functions.

File: compass/landice/mesh.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import numpy as np
+import jigsawpy
```
Comment (matthewhoffman) [inline compass/landice/mesh.py:2]: Now that this has moved here, it looks like the corresponding import in `compass/landice/tests/humboldt/mesh.py`can be safely deleted.

**Review by matthewhoffman — CHANGES_REQUESTED** [review]
@trhille , thanks for working on abstracting this out for general use.  I made two small comments, but otherwise this looks great.

@xylar, after I'm happy with those changes (and assuming CI still passes), are you ok with me merging myself?

**Comment by xylar** [comment]
Sure, feel free to merge, though ideally someone should set up and run the Humboldt test case one last time after all the changes come in.

File: compass/landice/mesh.py (hunk 2 above)
Comment (trhille) [reply compass/landice/mesh.py]: Renamed in [68a8205](https://github.com/MPAS-Dev/compass/pull/319/commits/68a82051daed908e1ebfbb3051b4a862c27aaa59)

**Comment by trhille** [comment]
I ran the Humboldt test case after addressing @matthewhoffman's requested changes in commit [68a8205](https://github.com/MPAS-Dev/compass/pull/319/commits/68a82051daed908e1ebfbb3051b4a862c27aaa59). The test passed and the mesh is identical to the previous version.

**Pull request merged**